apps/widget/package.json

```
@@ -5,6 +5,7 @@
   "dependencies": {
     "@ant-design/icons": "^4.5.0",
     "@mantine/core": "^4.1.0",
+    "@novu/bell": "workspace:^0.4.0-alpha.3",
```

```suggestion
    "@novu/bell": "^0.4.0-alpha.3",
```
You can just specify it like this, and run pnpm install from root I believe it will work.

---

Yea not sure where it came from, there is a fix on the next coming branch  🖲️